recipe/conda_build_config.yaml

```diff
@@ -395,7 +395,7 @@ geotiff:
 giflib:
   - 5.2
 glew:
-  - 2.0.0
+  - 2.1.0
```

Comment:
Shouldn't this be `2.1`?

Reply:
Yes

Reply:
That is fixed for future close outs of the bot

Reply:
#543